tbtaf/interpreter/TBATFInterpreterCLI.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import os as _os
+import sys
+import importlib.util
+
+currentPath = _os.path.dirname(_os.path.abspath(__file__))
+rootFolder = _os.path.abspath(_os.path.join(currentPath, ".."))
+scriptPath = _os.path.join(rootFolder, "orchestrator", "TBTAFOrchestrator.py")
+sys.path.append(rootFolder)
+spec = importlib.util.spec_from_file_location("TBTAFOrchestrator.py", scriptPath)
+modulo = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(modulo)
+
+class TBTAFInterpreterCLI(object):
+    def __init__(self):
+        self.myTBTAF = modulo.TBTAFOrchestrator()
+        self.command = ""
+        self.path = './temp.tbtaf'
+        try:
+            with open(self.path, 'w') as tempFile:
+                tempFile.write(self.command)
+                return
+            
+        except:
+            return
+        
+    def commandInterpreter(self):
+        if self.command == 'quit()' or self.command == 'Quit()' or self.command == 'QUIT()':

```

Comment:
As mentioned in the launcher script, please normalize to a single casing format and then compare in order to simplify the logic